components/workflow-mgt/org.wso2.carbon.identity.workflow.impl/src/main/java/org/wso2/carbon/identity/workflow/impl/WorkflowImplServiceImpl.java

```diff
@@ -174,6 +229,10 @@ public void deleteHumanTask(WorkflowRequest workflowRequest) throws WorkflowImpl
                 RemoteException | IllegalStateFault | IllegalArgumentFault e) {
             throw new WorkflowImplException("Error while deleting the human tasks of the request.");
```

Comment:
This has already been fixed with  https://github.com/wso2/carbon-identity/commit/a3d18711e29c26f313b8c8efd9c17302e3434eb1